### ISSUE TITLE
[Suggestion] Expose Resource Version in `getResourceUpdate` Response

### ISSUE THREAD

**Issue by amnotbananaama** [SpigotMC/XenforoResourceManagerAPI#51]
Hello, I am very excited to see an official API for Spigot resources! Thank you for your hard work getting this up and running. My suggestion is for the `getResourceUpdate` route to return a field, containing the version of the resource update. This would be the same version as specified on the resource's "Version History" page. See below for an example.

# Example
`GET` `?action=getResourceUpdate&id=17`
```json
{
  "id": "17",
  "resource_id": "2",
  "resource_version": "1.0",
  "title": "Added /lobby|/hub command",
  "message": "New commands.\n \n[LIST]\n[*]/hub\n[*]/lobby\n[/LIST]"
}
```
This field would be useful in several use cases that I am happy to elaborate on if needed. While adding any information to an API response is computationally expensive, I think this is an acceptable trade-off. Resource update versions cannot change (resource ID `17` will always have version `1.0`), and this information is already pre-existing on the resource's page, the API just needs to fetch it.

**Comment by amnotbananaama** [comment]
I ended up looking into this despite knowing very little about Xenforo and SQL. I managed to figure out the following.  
  
1. The version of a Resource Update is stored as the column `version_string` in the table `xf_resource_version`.

2. The API uses this internally to fetch the latest version of a Resource, filtering by the (already known) id of the latest update. https://github.com/SpigotMC/XenforoResourceManagerAPI/blob/e7db7b94436eed788a6ee53eba9b201e64460db6/src/support/Database.php#L211-L212  

So—in theory—we would be able to adapt that code into the internal function `_resource_update()`, using the id of the Resource Update from the request. Unfortunately, this is where I got stuck by my limited knowledge.  Could we do something like this perhaps?  
  
```diff
private function _resource_update($suffix) {
    return sprintf(
        "SELECT r.resource_update_id, r.resource_id, r.title, r.message
        FROM xf_resource_update r
+          INNER JOIN xf_resource_version rv 
+              ON r.resource_id = rv.resource_id         
        WHERE r.message_state = 'visible' %s",
        $suffix
    );
}
```
